Runtime/MonkeyConfig.cs

```diff
@@ -2,13 +2,28 @@
 // This software is released under the MIT License.
 
 using System;
+using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
+using TestHelper.Monkey.Matchers;
+using TestHelper.Monkey.Operators;
 using TestHelper.Monkey.Random;
 using TestHelper.Monkey.ScreenPointStrategies;
 using UnityEngine;
 
 namespace TestHelper.Monkey
 {
+    public struct MatcherOperatorPair
+    {
+        public IComponentMatcher _matcher;
+        public IOperator _operator;
+
+        public MatcherOperatorPair(IComponentMatcher matcher, IOperator @operator)
+        {
+            _matcher = matcher;
+            _operator = @operator;
+        }
+    }
```

Comment:
I think there are better ways to do it.

### Steps inside RunStep in this patch
1. collect components that can do any operation
2. pick a component within the components
3. pick a operation that can do on the picked component (it means available operation check get done twice in single RunStep call)

### Steps inside RunStep in my proposal
1. collect component and operator pairs that is available
2. pick a component and operator pair
3. do the operator on the component

And if it is possible, I want to inject the collect component operator selection strategy or picking strategy. Because I'm facing "back and forth" problem. In my application, many modals have 2 buttons; go to next, go to previous. Therefore the monkey almost back and forth between the modals in long time. Injecting the selection strategy or picking strategy can solve the problem by removing "go to previous" buttons from candidates if other button exist.